apps/docs/libraries/ts/ratelimit/override/set-override.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Set Override"
+description: "Sets an override for a ratelimit"
+---
+
+## Request
+ 
+
+ <ParamField body="identifier" type="string" required>
+Identifier of your user, this can be their userId, an email, an ip or anything else. Wildcards ( * ) can be used to match multiple identifiers, More info can be found at https://www.unkey.com/docs/ratelimiting/overrides#wildcard-rules
+</ParamField>
+
+<ParamField body="limit" type="number" required>
+How many requests may pass in a given window.
+</ParamField>
+
+<ParamField body="duration" type="number" required>
+The window duration in milliseconds.
+</ParamField>
+
+ <Warning>
+Either `namespaceId` or `namespaceName` is required. Not both.
+</Warning>
+
+<ParamField body="namespaceId" type="string" required>
+The id of the namespace. Either namespaceId or namespaceName must be provided
+</ParamField>
+
+<ParamField body="namespaceName" type="string" required>
+Namespaces group different limits together for better analytics. You might have a namespace for your public API and one for internal tRPC routes. Wildcards can also be used, more info can be found at https://www.unkey.com/docs/ratelimiting/overrides#wildcard-rules
+</ParamField>
+
+<ParamField body="async" type="boolean" default="false">
+Async will return a response immediately, lowering latency at the cost of accuracy.
+</ParamField>
+
+
+## Response
+
+<ResponseField name="result">
+  <Expandable title="properties" defaultOpen>
+    <ResponseField name="overrideId" type="string" required>
+    The id of the override that was set.
+    </ResponseField>
+  </Expandable>
+  </ResponseField>
+<RequestExample>
+
+```ts
+await unkey.setOverride({
+    identifier: "user_123",
+    limit: 10,
+    duration: 60000,
+    namespaceName: "email.outbound",
+    namespaceId: "rlns_1234",
```

Comment:
Show a "create" and an update with the namespaceId for example

Reply:
Should we be adding some kind of header text to the example tabs?